lib/internal/streams/duplex.js

```diff
@@ -42,8 +42,10 @@ ObjectSetPrototypeOf(Duplex.prototype, Readable.prototype);
 ObjectSetPrototypeOf(Duplex, Readable);
 
 {
+  const keys = ObjectKeys(Writable.prototype);
   // Allow the keys array to be GC'ed.
-  for (const method of ObjectKeys(Writable.prototype)) {
+  for (let i = 0; i < keys.length; i++) {
+    const method = keys[i];
     if (!Duplex.prototype[method])
       Duplex.prototype[method] = Writable.prototype[method];
   }
```

Comment:
nit, but I don't think this actually improves anything... quite the opposite

Reply:
https://github.com/nodejs/node/pull/41871#discussion_r800215084

This change is done for consistency and avoid mangled `Symbol.iterator`s, as it can have a significant impact on the performance.

Maybe @aduh95 can explain it better as they suggested the change.

Reply:
I don't think performance is relevant here though? This is a very cold path.

Reply:
Anyway, not a problem for me. Just a little weird IMHO. Not blocking.